.github/workflows/docker-release.yml

```diff
@@ -32,7 +74,17 @@ jobs:
           push: true
           file: Dockerfile
           tags: |
-            dsnp/example-client:${{ env.TAG }}
+            dsnp/example-client:${{ env.NEW_TAG_VERSION }}
+
+      - name: Build and push dsnp/example-client:latest
+        if: steps.set-result.outputs.result == 'true'
+        id: docker_build_example-client-latest
+        uses: docker/build-push-action@v2
```

Comment:
Does this trigger a rebuild?

Reply:
We added a conditional to both build blocks to prevent rebuilding